presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/IndexJoinOptimizer.java

```diff
@@ -326,7 +326,7 @@ public PlanNode visitFilter(FilterNode node, RewriteContext<Context> context)
         public PlanNode visitWindow(WindowNode node, RewriteContext<Context> context)
         {
             if (!node.getWindowFunctions().values().stream()
-                    .map(function -> function.getFunctionCall().getName())
+                    .map(f -> f.getFunction().getFunctionCall().getName())
```

Comment:
@erichwang: I think @maciejgrzybek fixed this in one of the follow-on commits that he's going to reorder and squash.